README.md

```diff
@@ -5,6 +5,21 @@ O3DE (Open 3D Engine) is an open-source, real-time, multi-platform 3D engine tha
 ## Contribute
```

Comment:
Looks like this file might be unintentional in your commit?
I think you need to kick this file from you PR, replace with updates in our read me <dccsi>\readme.md?

I love the ideas of readme with notes, impactful changes, and such.
This file is however the root readme.md for o3de:
https://github.com/o3de/o3de/blob/development/README.md

I think we can just follow the pattern, but log our own readme's:
1. when is is worth altering the core o3de readme?
2. should we just extend the <dccsi>\readme?
3. per-package shared readme at root of this API package <dccsi>\azpy\readme.md?
4. or nested per-module nested <dccsi>\azpy\o3de\readme.md files?

Reply:
I did not create this file- it was included automatically. I assumed it was part of some sort of templated system?